reactfront/src/components/panels/graph-panel.js

```diff
@@ -170,7 +171,9 @@ export function Datalogger({
       setWindow([Math.min(left, right - 0.01), right])
     }
 
-    const getXFromEvent = e => e.clientX - svgRef.current?.getBoundingClientRect().left
+    const getXFromEvent = function(e) {
```

Comment:
Could we keep this as `e => {` instead of `function(e) {` for consistent code style?